deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -175,6 +175,12 @@ spec:
                                     type: string
                                 type: object
                             type: object
+                          endpoint_cert_duration:
```

Comment:
We might want to add a pattern expression here as well like they do for Prometheus CRD for the retention value:

https://github.com/prometheus-operator/prometheus-operator/blob/main/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml#L5538-L5544

Reply:
For what I understand, the duration can be only expressed in hours (based on the code here https://github.com/cert-manager/cert-manager/blob/v1.2.0/pkg/apis/certmanager/v1/const.go#L26). So I'll use a regex only for that.

Reply:
I got an email referencing a comment here about "can be only expressed in hours", but maybe it was deleted? In case there is doubt about the type, it is here: https://github.com/cert-manager/cert-manager/blob/master/deploy/crds/crd-certificates.yaml#L91 

"Value must be in units accepted by Go time.ParseDuration https://golang.org/pkg/time/#ParseDuration"

Reply:
Yes Chris, thanks for checking... I added a pattern to match that type.